File: StudentCredit.Application/Banks/Dtos/BankContractDto.cs
using FileStorageNetCore.Models;

namespace StudentCredit.Application.Banks.Dtos
{
    public class BankContractDto
    {
        public int? Id { get; set; }
        public string Number { get; set; }
        public DateTime Date { get; set; }
        public ContractType Type { get; set; }
        public AttachedFile File { get; set; }
    }
}

File: WebApplication/src/modules/bank/model/bank-contract.dto.ts
import { AttachedFile } from "src/infrastructure/models/attached-file.model";
import { ContractType } from "../enums/contract-type.enum";

export class BankContractDto {
  id: number;
  number: string;
  date: Date;
  type: ContractType;
  file: AttachedFile;

  constructor(type: ContractType) {
    this.type = type;
  }
}